IBPSA/Resources/C-Sources/fileWriterFree.c

```diff
@@ -51,10 +51,11 @@ void fileWriterFree(void* ptrFileWriter){
   /* If this FileWriter writes in a CombiTimeTable format, prepend the required header
   now that we know how many lines have been written. */
   if (ID->isCombiTimeTable){
-    char buf[255];
-    sprintf(buf,"#1\ndouble csv(%i,%i)\n",ID->numRows,ID->numColumns);
+    char* buf;
+    asprintf(&buf,"#1\ndouble csv(%i,%i)\n",ID->numRows,ID->numColumns);
```

Comment:
Trying to figure out why unit tests fail. I find online that `asprintf` is a GNU extension. Not. sure what that means,  but is it supported by the docker/travis?

Reply:
I was wondering that too, but for jmodelica the compiler is in the docker and hence this should work on travis as it does locally where I used the same image.
I will change this back and see if this is the reason, but I doubt it is.

Reply:
Perhaps we should add something to Buildingspy where the Dymola error message is included when the Simulation fails?

Reply:
@Mathadon : `asprintf` is not the problem as the test https://travis-ci.org/ibpsa/modelica-ibpsa/builds/520953878 also fails. I probably have to look into it next week when back from travel.  Branch`issue1114_jsonWriter_testing` is set up to run just this test.